test/e2e/tsconfig.e2espec.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016"
```

Comment:
can we try es2018 to match des4.

Reply:
Changed to 2018, nothing changed from what I can see